bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -491,7 +493,7 @@ private void refreshAll() {
         ptA[index++] = BasePositionType.instance;
         for (GuideProbe guider : guidersList) {
             final boolean guiderNotIllegal = !guider.equals(illegal);
-            ptA[index++] = new GuidePositionType(guider, guiderNotIllegal);
+            ptA[index++] = new GuidePositionType(guider, guiderNotIllegal, _w.positionTable);
```

Comment:
Unrelated to this PR, I don't follow the logic that sets `illegal` and therefore `guiderNotIllegal` (which seems better named `guiderIsLegal` or something).  My issue is that there can be multiple "illegal" guiders and yet we only keep up with the last one we saw?  (See line 481.)  It seems a bit random whether `guiderNotIllegal` if there is more than one illegal guider.  Does it make sense to you?